### PR TITLE
Locate errors

### PR DESCRIPTION
**Pull request by Acentelles**

### CODE REVIEW

**Comment by cwgoes** [comment]
What's remaining here? Looks like this needs a rebase to start.

**Comment by Acentelles** [comment]
Yes! And some tests are still not passing

**Comment by cwgoes** [comment]
Note that if this is still a work-in-progress, please mark it as "draft" using Github (no need to mark it in the title if you do that).

**Comment by cwgoes** [comment]
Just checking, is this R4R now?

**Comment by Acentelles** [comment]
Yes, it's ready to review

File: library/Frontend/package.yaml [hunk 1]
```diff
@@ -26,6 +26,7 @@ default-extensions:
 
 dependencies:
 - base >= 4.7 && < 5
+- protolude
```
Comment (mariari) [inline library/Frontend/package.yaml:29]: Should probably remove this

**Review by mariari — APPROVED** [review]
Looks fine

**Pull request merged**